packages/squiggle-lang/__tests__/TS/PointSet_test.ts

```diff
@@ -0,0 +1,49 @@
+import { errorValueToString } from "../../src/js/index";
+import { testRun, failDefault, expectErrorToBeBounded } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Mean of mixture is weighted average of means", () => {
+  test("mx(beta(a,b), lognormal(m,s), [x,y])", () => {
+    fc.assert(
+      fc.property(
+        fc.float({ min: 1e-1 }), // alpha
+        fc.float({ min: 1 }), // beta
+        fc.float(), // mu
+        fc.float({ min: 1e-1 }), // sigma
+        fc.float({ min: 1e-7 }),
+        fc.float({ min: 1e-7 }),
+        (a, b, m, s, x, y) => {
+          let squiggleString = `mean(mx(beta(${a},${b}), lognormal(${m},${s}), [${x}, ${y}]))`;
+          let res = testRun(squiggleString);
+          switch (res.tag) {
+            case "Error":
+              expect(errorValueToString(res.value)).toEqual(
+                "<I wonder if test cases will find this>"
+              );
+              break;
+            case "Ok":
+              let weightDenom = x + y;
+              let betaWeight = x / weightDenom;
+              let lognormalWeight = y / weightDenom;
+              let betaMean = 1 / (1 + b / a);
+              let lognormalMean = m + s ** 2 / 2;
```

Comment:
I think we should just use @genType or similar on the Symbolic Dists library, and use that to find the means of these.

Reply:
I'm reasonably confident that jstat authors are reading the same wikipedia articles I'm reading, based on some rescript tests in another file. It's a trivial decision. What I like about this is if dependabot sends us a new version of Jstat and they accidentally pushed something that diverges from wikipedia, we'll catch it if and only if we recompute in test like this.